mlir-sycl/lib/Dialect/SYCL/IR/SYCLOps.cpp

```diff
@@ -210,23 +210,37 @@ void SYCLConstructorOp::getEffects(
 #define GET_OP_CLASSES
 #include "mlir/Dialect/SYCL/IR/SYCLOps.cpp.inc"
 
-sycl::SYCLIDGetOp sycl::createSYCLIDGetOp(TypedValue<MemRefType> id,
-                                          unsigned index, OpBuilder builder,
-                                          Location loc) {
+SYCLIDGetOp sycl::createSYCLIDGetOp(TypedValue<MemRefType> id, unsigned index,
```

Comment:
[Suggestion]: If you wrap  the function definitions in  `namespace sycl { }` then you can remove `sycl::`

Reply:
done.